### PR TITLE
OG image

### PR DESCRIPTION
**Pull request by matamalaortiz**
Added an OG Image.

### CODE REVIEW

File: website/npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,26 @@
+0 info it worked if it ends with ok
```
Comment (cvalenzuela) [inline website/npm-debug.log]: I think you should not have committed this file.
Can you remove it?

**Comment by matamalaortiz** [comment]
deleted and ignored.

**Comment by handav** [comment]
Just want to check that that's not a real api key in siteconfig?

**Comment by cvalenzuela** [comment]
It is the api key for the search engine and it is public. no worries! See #4

**Comment by handav** [comment]
Great, thanks! 👍

**Pull request merged**